src/catalog/lib.rs

```diff
@@ -12,9 +12,11 @@ use std::fmt;
 use std::path::Path;
 use std::sync::{Arc, Mutex, MutexGuard};
 
+use chrono::Utc;
 use failure::bail;
 use lazy_static::lazy_static;
 use log::{info, trace};
+use rand::random;
```

Comment:
Rust convention is to not import standalone functions, and instead use the module-qualified path at the point of use: `rand::random`.